cfgov/sheerlike/templates.py

```diff
@@ -4,16 +4,9 @@
 
 
 def _convert_date(date, tz):
-    if date:
-        if isinstance(date, basestring):
-            date = parser.parse(date,
-                                default=datetime.datetime.today().replace(day=1))
-        if type(date) in [datetime.datetime, datetime.date]:
-            if isinstance(date, datetime.datetime):
-                pytzone = timezone(tz)
-                if date.tzinfo:
-                    date = date.replace(tzinfo=None)
-                date = pytzone.localize(date)
+    if date and isinstance(date, basestring):
+        date = parser.parse(date,
+                            default=datetime.datetime.today().replace(day=1))
     return date
 
```

Comment:
I don't think we have to remove the timezone logic entirely, it's just that we shouldn't be stripping the timezone info.  Here's what I had done in my former PR: ```def _convert_date(date, tz):
    if date:
        if isinstance(date, basestring):
            date = parser.parse(date,
                                default=datetime.datetime.today().replace(day=1))
        if type(date) in [datetime.datetime, datetime.date]:
            if isinstance(date, datetime.datetime):
                pytzone = timezone(tz)
                if not date.tzinfo:
                    date = pytzone.localize(date)
                else:
                    date = date.astimezone(pytzone)

    return date
```

If we are going to get rid of it though, we just need to make sure other functions that call this don't break. `when` in `query.py` relies on this function.